symplyphysics/laws/electricity/transmission_lines/ratio_of_average_power_to_incident_power_from_reflection_coefficient.py

```diff
@@ -0,0 +1,36 @@
+from sympy import Eq, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+    dimensionless,
+    convert_to_float,
+)
+
+# Description
+## The reflection coefficient is equal to the ratio of the reflected wave power to the incident wave power.
+## Knowing the average power delivered to the load and the incident power, it is possible to calculate the absolute value reflection coefficient.
+
+## Law is: Pa / Pi = 1 - G^2, where
+## Pa - the average power delivered to the load,
+## Pi - the incident power,
+## K - the absolute value reflection coefficient.
+
+absolute_reflection_coefficient = Symbol("absolute_reflection_coefficient", dimensionless, rel=True)
```

Comment:
Why `rel=True`?

Reply:
> Why `rel=True`?

Because the reflection coefficient modulus cannot be imaginary.

Reply:
It's `real` as far as I remember.

Reply:
> It's `real` as far as I remember.

Is it real that the reflection coefficient module can contain an imaginary part? No, that doesn't happen.